code/modules/mob/living/simple_animal/hostile/ordeal/black/_black.dm

```diff
@@ -0,0 +1,333 @@
+	minimum_distance = 1
+	ranged = TRUE
+	attack_verb_continuous = "bashes"
+	attack_verb_simple = "bash"
+	projectiletype = /obj/item/ammo_casing/caseless/ego_adoration/dot
+	projectilesound = 'sound/effects/attackblob.ogg'
+	attack_sound = 'sound/effects/attackblob.ogg'
+
+//Soulmate
+/mob/living/simple_animal/hostile/ordeal/echo/soulmate
+	icon_state = "soulmate_echo"
+	icon_living = "soulmate_echo"
+	melee_damage_type = BLACK_DAMAGE
+	melee_damage_lower = 80
+	melee_damage_upper = 80
+	damage_coeff = list(RED_DAMAGE = 0.3, WHITE_DAMAGE = 0.5, BLACK_DAMAGE = 0.3, PALE_DAMAGE = 0.5)
+
+	rapid = 1
+	rapid_fire_delay = 2
+	ranged_cooldown_time = 7
+	ranged = TRUE
+	attack_verb_continuous = "rips"
+	attack_verb_simple = "rip"
+	projectiletype = /obj/projectile/ego_bullet/gunblade
+	projectilesound = 'sound/weapons/ionrifle.ogg'
+	attack_sound = 'sound/weapons/blade1.ogg'
+
+//spooner
+/mob/living/simple_animal/hostile/ordeal/echo/spooner
+	icon_state = "homing_echo"
+	icon_living = "homing_echo"
+	melee_damage_type = BLACK_DAMAGE
+	melee_damage_lower = 209
+	melee_damage_upper = 209
+	damage_coeff = list(RED_DAMAGE = 1.2, WHITE_DAMAGE = 0.7, BLACK_DAMAGE = 0.5, PALE_DAMAGE = 0.9)
+
+	attack_verb_continuous =  "smacks"
+	attack_verb_simple = "smacks"
+	attack_sound = 'sound/weapons/ego/hammer.ogg'
+
+
+/*************************************************************************************************
+**************************************************************************************************
+**************************************************************************************************
+						Superbosses
+**************************************************************************************************
+**************************************************************************************************
+*************************************************************************************************/
+
+//Distorted form
+/mob/living/simple_animal/hostile/ordeal/echo/distorted
+	icon_state = "distortion_echo"
+	icon_living = "distortion_echo"
+	melee_damage_type = RED_DAMAGE
+	melee_damage_lower = 80
+	melee_damage_upper = 80
+	rapid_melee = 1
+	damage_coeff = list(RED_DAMAGE = 0.2, WHITE_DAMAGE = 0.3, BLACK_DAMAGE = 0.2, PALE_DAMAGE = 0.5)
+
+	attack_verb_continuous = "pulverizes"
+	attack_verb_simple = "pulverize"
+	attack_sound = 'sound/weapons/ego/hammer.ogg'
+
+
+/mob/living/simple_animal/hostile/ordeal/echo/distorted/AttackingTarget()
+	. = ..()
+	if(!.)
+		return
+	if(!ishuman(target))
+		return
+	var/mob/living/H = target
+	for(var/damage_type in list(WHITE_DAMAGE, BLACK_DAMAGE, PALE_DAMAGE))
+		H.apply_damage(melee_damage_lower, damage_type, null, H.run_armor_check(null, damage_type))
+
+
+/mob/living/simple_animal/hostile/ordeal/echo/distorted/attacked_by(obj/item/I, mob/living/user)
+	//the block
+	if(prob(30))
+		damage_coeff = list(RED_DAMAGE = 0.08, WHITE_DAMAGE = 0.12, BLACK_DAMAGE = 0.08, PALE_DAMAGE = 0.2)
```

Comment:
```suggestion
		ChangeResistances(0.08, 0.12, 0.08, .2)
```

Reply:
Thought I already did this, maybe I just didn't push it